src/3rdparty/walkontable/test/spec/selection.spec.js

```diff
@@ -155,11 +155,19 @@ describe('Walkontable.Selection', () => {
 
     wt.scrollViewportVertically(17);
     wt.draw();
-    expect(wt.wtTable.getFirstVisibleRow()).toEqual(10);
+
+    let expectedFirstVisibleRow = 10;
+    if (wt.wtSettings.getSetting('scrollbarHeight') >= 17) {
```

Comment:
And on Windows in MSEdge is `16px`. Maybe it'll be better to calculate `expectedFirstVisibleRow` basing on container height and rows heights?

Reply:
Thanks for the suggestion. I have pushed a soution with even cleaner solution in the commit https://github.com/handsontable/handsontable/pull/6187/commits/d32db4736a774afb86eeb081db38eefc716cb512. 

The container size is normalized to have the same inner size disregarding of the scrollbar size.